src/Subscription.js

```diff
@@ -44,15 +43,15 @@ export default class Subscription extends EventEmitter {
 
         // Check that multiple resend options are not given
         let resendOptionCount = 0
-        if (this.options.resend_from != null) {
-            if (!(this.options.resend_from instanceof MessageRef)) {
-                throw new Error(`resend_from option needs to be a MessageRef: ${this.options.resend_from}`)
-            }
+        if (this.options.from != null) {
```

Comment:
Looks like the counter thing can be removed and code reduced to equivalent:
```
if (this.options.from != null && this.options.last != null) {
    throw new Error(`Multiple resend options active! Please use only one: ${JSON.stringify(this.options)}`)
}
```

Might also be worth checking for invalid condition 
`this.options.from == null && this.options.to != null`.